src/views/shared/SecurityBeacon.js

```diff
@@ -15,6 +15,18 @@ define(['okta', 'util/Animations'], function (Okta, Animations) {
   var _ = Okta._,
       $ = Okta.$;
 
+  // TODO(OKTA-118980): Server should send description as part of appState.
+  function getSecurityImageDescriptionFromUrl(imgSrc) {
+    var parser = document.createElement('a');
+    parser.href = imgSrc;
+
+    // Get stuff between the last slash, and the first period after that slash.
+    // (if the string ends with a slash, use the second-to-last slash instead.)
+    var match = /^.*\/([^.]+).*\/?$/.exec(parser.pathname);
+    var basename = (match.length >= 2) ? match[1] : null;
+    return basename ? (basename + ' security image') : '';
```

Comment:
we need to localize this string... this is tricky since the image filename is what we are using to describe the image... so maybe we need something like:
```
securityimage.aria.text = {0} security image
securityimage.beer = beer
securityimage.flower = flower
...
```